db/1.10/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -757,7 +755,7 @@ When working with pooling profiles and the Query Single operation, the connectio
 `<db:stored-procedure>`
 
 
-Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, streaming is applied automatically to avoid preemptive consumption of the results, which may lead to performance and memory issues.
+Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead results are automatically streamed to prevent performance and memory issues. This behavior means that pages of _fetchSize_ rows are loaded lazily when needed. If the *Stored procedure* operation performs inside a transaction (for example a *Try* scope component), and that transaction is closed before consuming the data, accessing the results which haven't been loaded fail.
```

Comment:
```suggestion
Invokes a stored procedure on the database.  When the stored procedure returns one or more ResultSet instances, results are not read all at once. Instead, results are automatically streamed to prevent performance and memory issues. This behavior means that pages of `_fetchSize_` rows are loaded lazily when needed. If the *Stored procedure* operation performs inside a transaction (for example, in a *Try Scope* component), and that transaction is closed before consuming the data, accessing the results that haven't been loaded fails.
```